util/process_wrapper/main.rs

```diff
@@ -61,10 +61,19 @@ impl fmt::Display for ProcessWrapperError {
 
 impl std::error::Error for ProcessWrapperError {}
 
+macro_rules! log {
+    ($($arg:tt)*) => {
+        if std::env::var("RULES_RUST_PROCESS_WRAPPER_DEBUG").is_ok() {
```

Comment:
Super nitty, but this tests precisely for "is this set", rather than that it was set _and valid UTF-8_ :)

```suggestion
        if std::env::var_os("RULES_RUST_PROCESS_WRAPPER_DEBUG").is_some() {
```